__tests__/app.test.js

```diff
@@ -47,3 +47,60 @@ describe("/api", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("should return an array of comments with correct properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        response.body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
We should be more specific here and specify that the article_id should always be 1